src/__test__/App.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import App from '../components/App';
+
+
+describe('App', () => {
+  test('should match snapshot', () => {
```

Comment:
use the word `it` instead of `test` for each test i.e.
`it(your test)`

Reply:
@MuhanguziDavid this has been resolved

Reply:
@frankopkusianwar , I meant that you needed to change the word `test` to `it` literally changing only that word. Leave the rest as it was.